Library/src/main/java/info/metadude/android/typedpreferences/BasePreference.java

```diff
@@ -0,0 +1,30 @@
+package info.metadude.android.typedpreferences;
+
+import android.content.SharedPreferences;
+
+class BasePreference {
+
+    protected final SharedPreferences mPreferences;
+    protected final String mKey;
+
+    public BasePreference(final SharedPreferences preferences, final String key) {
```

Comment:
How do you suggest the two parameters constructor can be hidden? It is currently accessible from the test classes.